docs/source/tools/codegen.rst

```diff
@@ -7,37 +7,31 @@ DAML codegen
 Introduction
 ============
 
-You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates. These classes incorporate all boilerplate code for constructing corresponding ledger :ref:`com.daml.ledger.api.v1.CreateCommand` and :ref:`com.daml.ledger.api.v1.ExerciseCommand`.
+You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates.
+These classes incorporate all boilerplate code for constructing corresponding ledger ``com.daml.ledger.api.v1.CreateCommand``
+and ``com.daml.ledger.api.v1.ExerciseCommand``.
 
 Running the DAML codegen
 ========================
 
 The basic command to run the DAML codegen is::
 
-  $ daml codegen [java|scala] [options]
+  $ daml codegen [java|scala|js] [options]
 
 There are two modes:
 
-- command line configuration, specifying **all** settings in the command line
+- Command line configuration, specifying **all** settings in the command line (all codegens supported)
 
-- project file configuration, specifying **all** settings in the ``daml.yaml``
+- Project file configuration, specifying **all** settings in the ``daml.yaml`` (currently **Java** and **Scala** only)
 
 Command line configuration
 --------------------------
 
-Help for **DAML to Java** codegen::
+Help for each specific codegen::
 
-  $ daml codegen java --help
+  $ daml codegen [java|scala|js] --help
 
-Help for **DAML to Scala** codegen::
-
-  $ daml codegen scala --help
-
-Both **DAML to Java** and **DAML to Scala** take the same set of configuration settings::
-
-    Usage: codegen [options] <DAR-file[=package-prefix]>...
-
-    Code generator for the DAML ledger bindings.
+**Java** and **Scala** take the same set of configuration settings::
```

Comment:
I'm not sure it is clear that we're referring to the codegens to Java/Scala here.